### DISCUSSION TITLE
V6: How to Authenticate from existing session

### DISCUSSION THREAD

**Discussion by simonjcarr** [adonisjs/core#4738] [Help]
I have an authVerify method in my users_controller, but it is returning 401
```js
public async authVerify({ auth, response }: HttpContext) {
    try {
      await auth.use('web').authenticate()
    } catch (error) {
      console.log('Error', error)
    }
    const user = auth.use('web').user!
    console.log(user)
    return response.json(user)
  }
```
The idea is that when the user refreshes the Frontend App, I have a boot function that makes a call to an endpoint
```js
router
  .group(() => {
    router.post('/auth_check', [UsersController, 'authVerify']).as('authVerify')
  })
  .prefix('api/v1')
  .use([middleware.auth()])
```
So that the user record can be returned based on the session in the browser.
I am getting a 401 error, I am refreshing the browser a few seconds after the logging in, so the token should still be valid.
```bash
auth.js?t=1727347826163:9
       POST http://localhost:3333/api/v1/auth_check 401 (Unauthorized)
```
The request is not getting to the controller, and is being blocked by the auth middleware.

**Comment by thetutlage** [comment]
Couple of questions!

- Are you using session auth?
- If yes, is your frontend app forwarding cookies when making the AJAX/Fetch request?

**Comment by simonjcarr** [comment]
Thank you, the issue was related to cores.
